# verbatim (language="comment",  text=" relative time from beginning")
float64 relative_time

float64 x

float64 y

float64 z

float64 theta

geometry_msgs/Quaternion orientation1

# verbatim (language="comment",  text=" accumulate s from beginning")
float64 s

# verbatim (language="comment",  text=" turn right is -, turn left is +")
float64 kappa

float64 dkappa

float64 v

float64 a
